src/code/z_parameter.c

```diff
@@ -169,15 +169,18 @@ static Gfx sSetupDL_80125A60[] = {
 };
 
 // original name: "alpha_change"
-void Interface_ChangeAlpha(u16 alphaType) {
-    if (alphaType != gSaveContext.unk_13EA) {
-        osSyncPrintf("ＡＬＰＨＡーＴＹＰＥ＝%d  LAST_TIME_TYPE=%d\n", alphaType, gSaveContext.unk_13EE);
-        gSaveContext.unk_13EA = gSaveContext.unk_13E8 = alphaType;
-        gSaveContext.unk_13EC = 1;
+void Interface_SetHudVisibility(u16 hudVisibility) {
+    if (hudVisibility != gSaveContext.hudVisibility) {
+        osSyncPrintf("ＡＬＰＨＡーＴＹＰＥ＝%d  LAST_TIME_TYPE=%d\n", hudVisibility, gSaveContext.prevHudVisibility);
+        gSaveContext.hudVisibility = gSaveContext.nextHudVisibility = hudVisibility;
+        gSaveContext.hudVisibilityCounter = 1;
     }
 }
 
-void func_80082644(PlayState* play, s16 alpha) {
+/**
+ * Slowly restore buttons to the hud unless a button is disabled
```

Comment:
Not sure on my wording but this may be an improvement still:
```c
 * Sets the button alphas to be dimmed for disabled buttons, or to the requested alpha for non-disabled buttons.
```